src/amo/components/AddonMeta.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import { compose } from 'redux';
 
+import Link from 'amo/components/Link';
 import translate from 'core/i18n/translate';
+import { isAddonAuthor } from 'core/utils';
+import type { AddonType } from 'core/types/addons';
 import LoadingText from 'ui/components/LoadingText';
 import Rating from 'ui/components/Rating';
 
 import 'amo/css/AddonMeta.scss';
 
 
+type PropTypes = {
+  addon: AddonType | null,
+  i18n: Object,
+  userId: number | null,
```

Comment:
Not sure we have discussed this yet (at least not with me), but Flow users tend to use `?number` to mark a nullable/maybe value: https://flow.org/en/docs/types/maybe/.

Reply:
Also I would go for strict typing if possible.

Reply:
I guess it's up to the design of the component. If it were `userId?` then `undefined` would be allowed. I think allowing `undefined` is risky and we should avoid it whenever possible. I like using `null` values to indicate that the property value is still loading.